django_mojeid/models.py

```diff
@@ -47,7 +48,7 @@ def __init__(self, *args, **kwargs):
 
         # Generate default salt
         if not 'salt' in kwargs:
-            kwargs['salt'] = os.urandom(30).encode('base64')[:-1]
+            kwargs['salt'] = base64.b64encode(os.urandom(30))
```

Comment:
sel jsem podle tohoto tracebacku co byl v tasku, jak to mam otestovat na produkci netusim, opravil jsem proste tuhle chybu, jestli je tam i nejaka jina, o te nevim